### PR TITLE
(#194) changed Require parameter to strings List

### LINKED ISSUE #194: The "Require" parameter option has not the correct type
In the current implementation the `Require` property in `AsciiDoctorJRunnerSettings` is implemented as `bool` (switch) but it has to be implemented as an `List<string>`, so that required extension libraries can be added via that parameter. 

Possible values for example are:

- `asciidoctor-diagram`
- `asciidoctor-epub3`
- `asciidoctor-revealjs`

The values then should be added as comma-separated list like `--require "asciidoctor-diagram,asciidoctor-epub3,asciidoctor-revealjs"` or as multiple parameters like  `--require asciidoctor-diagram --require asciidoctor-epub3 --require asciidoctor-revealjs`

### PR DESCRIPTION
**Pull request by nils-a**
So that required extension libraries can be added via that parameter.

fixes #194

### CODE REVIEW

**Comment by nils-a** [comment]
@mcpride, would you be in a position to take this for a spin?

File: src/Cake.AsciiDoctorJ.Tests/SettingsExtensionsTests.cs [hunk 1]
```diff
@@ -71,7 +71,7 @@ private IEnumerable<(Action<AsciiDoctorJRunnerSettings>, string)> GetTestData()
             yield return (s => s.WithVerbose(), "--verbose");
             yield return (s => s.WithTimingsMode(), "--timings");
             yield return (s => s.WithSectionNumbers(), "--section-numbers");
-            yield return (s => s.WithRequire(), "--require");
+            yield return (s => s.WithRequire("asciidoctor-epub3"), "--require asciidoctor-epub");
```
[the remark this answers is not in the document]
Comment (mcpride) [reply src/Cake.AsciiDoctorJ.Tests/SettingsExtensionsTests.cs:74]: In the test `actual.ShouldContain(excpectedParam)` is used, which is not sharp enough for that mistake, so the test doesn't fail!

**Comment by mcpride** [comment]
I tested it also in real wis pandoc v2.19.2 and v3.0.1 and it does what it should! 👍

**Comment by mcpride** [comment]
I tested it in real with AsciiDoctorJ 2.5.7 and it it works as expected!

File: src/Cake.AsciiDoctorJ.Tests/SettingsExtensionsTests.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (nils-a) [reply src/Cake.AsciiDoctorJ.Tests/SettingsExtensionsTests.cs:74]: 👀 Oho. Seems my tests are a bit "lax" :-D 
Good Catch.

**Comment by nils-a** [comment]
Thanks, @mcpride.
I will fix the test, rebase and then create a new release.

**Pull request merged**